src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -10,6 +10,10 @@
          https://console.developers.google.com/apis/api/{@class.apiName}
       2. Get access keys for your application. See
          https://developers.google.com/api-client-library/javascript/start/start-js#get-access-keys-for-your-application
+      @if class.auth.instructionsUrl
+        3. For additional information on making an authenticated call, see
```

Comment:
I think the most useful information was to point out that the sample should be run on a server - it won't run if just put into `index.html` and loaded on a browser. Also, the general instructions on how to set up the OAuth callback seemed useful.

After thinking about it a bit more, I'm satisfied with changing the wording to 'authentication'. Done.